frontend/src/views/ProgressOverview.js

```diff
@@ -72,7 +72,6 @@ class ProgressOverview extends Component {
         label: `${course.name} | ${course.number} | ${course.id}`,
         name: course.name,
         number: course.number,
-        id: course.id,
```

Comment:
@OmarIthawi Why do we remove this?

Reply:
the pipleline breaks due to having duplicate `id: course.id` in two lines:

![image](https://user-images.githubusercontent.com/645156/121869058-51c27e80-cd0a-11eb-8fdb-47edb4a7d613.png)